usaspending_api/api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -103,4 +103,4 @@ This endpoint returns data corresponding to the latest data dictionary csv file.
 + `total_rows`: 393 (required, number)
 + `total_columns`: 12 (required, number)
 + `total_size`: `119.32KB` (required, string)
-+ `download_location`: `http://files-nonprod.usaspending.gov/docs/DATA+Transparency+Crosswalk.xlsx` (required, string)
++ `download_location`: `http://files.usaspending.gov/docs/DATA+Transparency+Crosswalk.xlsx` (required, string)
```

Comment:
```suggestion
+ `download_location`: (required, string)
```

In retrospect, I don't think we really need to be testing the exact URL. I would advise NOT specifying value here, as it's just a contract.

Reply:
I like that more. Will add that into my changes.

Reply:
On that note actually, it might be weird that everything else says the value and type next to it. Thoughts?

Reply:
We've had this debate for a while at this point, and TBH I forgot the resolution, but I would drop all the values. We aren't promising consumers that the file will be exactly 119.32 Kilobytes. @ebdabbs, you agree?

Reply:
For the time being I have put it back except correcting it to `https` instead of `http`. I agree it is weird having exact values, but don't want just download_location to be empty.